// Copyright 2024 MozoLM Authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// Message encapsulating configuation options for NGramWordFst model.

syntax = "proto3";

package mozolm;

option java_outer_classname = "NGramWordFstOptionsProto";

// Next available ID: 2
message NGramWordFstOptions {
  // Maximum number of states to cache. Uses default if not set.
  int64 max_cache_size = 1;
}
